Session.php

```diff
@@ -187,7 +192,7 @@ public function destroySession($id)
     public function gcSession($maxLifetime)
     {
         $this->db->getCollection($this->sessionCollection)
-            ->remove(['expire' => ['$lt' => time()]]);
+            ->remove(['expire' => ['$lt' => new \MongoDate()]]);
```

Comment:
I don't get: if we rely on MongoDB internal mechanism for garbage collection, why manually do the same thing?